pyproject.toml

```
@@ -0,0 +1,125 @@
+[project]
+name = "OpenImageIO"
+# The build backend ascertains the version from the CMakeLists.txt file.
+dynamic = ["version"]
+description = "Reading, writing, and processing images in a wide variety of file formats, using a format-agnostic API, aimed at VFX applications."
+authors = [
+    {name = "Larry Gritz", email = "lg@larrygritz.com"},
+    {name = "OpenImageIO Contributors", email = "oiio-dev@lists.aswf.io"}
+]
+maintainers = [
+    {name = "OpenImageIO Contributors", email="oiio-dev@lists.aswf.io"},
+]
+readme = "README.md"
+license = {text = "Apache-2.0"}
+classifiers = [
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "License :: OSI Approved :: Apache Software License",
+    "Topic :: Multimedia :: Graphics",
+    "Topic :: Multimedia :: Video",
+    "Topic :: Multimedia :: Video :: Display",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+requires-python = ">= 3.8"
+
+[project.urls]
+Homepage = "https://openimageio.org/"
+Source = "https://github.com/AcademySoftwareFoundation/OpenImageIO"
+Documentation = "https://openimageio.readthedocs.io"
```

```suggestion
Documentation = "https://docs.openimageio.org"
```
I'm not sure if OIIO have fully switched to their vanity URL yet though.

---

I think the new URLs all work. I believe this can be verfied with any browser. But it does look like there are a few places in the docs where we use the rtd URL instead of the openimageio.org link. Those should be updated.